docs/platforms/javascript/guides/angular/features/component-tracking.mdx

```diff
@@ -13,25 +13,14 @@ To set up component tracking, you need to configure performance monitoring. For
 
 To track your components as part of your transactions, use any (or a combination) of the following options.
 
-## Using `TraceDirective`
+## Using the `trace` directive
```

Comment:
Should we not let this be 
```suggestion
## Using `TraceDirective`
```

for consistency, since below we also use `Using TraceClass` and `Using  TraceMethod` ?

Reply:
My train of thought was that users would "use" the directive mostly by specifying `trace='myComponent'`. The only time they'd actually need `TraceDirective` (the class) is when registering/importing it in their module. So there's a small difference to the decorators semantically. But I don't have strong feelings; we can also go back to the old heading

Reply:
ah I see, makes sense then - I don't know about angular wording, all good then!